src/main/scripts/setup/hydra_setup_script.py

```diff
@@ -0,0 +1,75 @@
+import os
+import argparse
+import setup_helpers
+from fabric.api import *
+import ntpath
+
+parser = argparse.ArgumentParser(description='Hydra setup script')
+parser.add_argument('--setup_ips_dir', '-i', type=str, default=os.environ['HOME'], help='Directory where mesos_masters_ips, mesos_slaves_ips and mesos_all_ips files are located. Default is /home/$USER')
+
+parser.add_argument('--dst_user_name', '-u', type=str, default="plumgrid", help='Destination user name')
+parser.add_argument('--dst_work_dir', '-w', type=str, default="/home/plumgrid", help='Destination work directory. All contents will be uploaded here.')
```

Comment:
Again default='/home/plumgrid' is not good, use the username extracted from environment.